jsapp/js/components/common/koboRange.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import bem, {makeBem} from 'js/bem'
+import './koboRange.scss'
+
+bem.KoboRange = makeBem(null, 'kobo-range')
+
+bem.KoboRange__values = makeBem(bem.KoboRange, 'values', 'div')
+bem.KoboRange__maxValue = makeBem(bem.KoboRange, 'max-value', 'span')
+bem.KoboRange__currentValue = makeBem(bem.KoboRange, 'current-value', 'span')
+bem.KoboRange__number = makeBem(bem.KoboRange, 'number', 'span')
+bem.KoboRange__unit = makeBem(bem.KoboRange, 'unit', 'span')
+
+
+bem.KoboRange__progress = makeBem(bem.KoboRange, 'progress', 'div')
+bem.KoboRange__input = makeBem(bem.KoboRange, 'input', 'input')
+
+export enum KoboRangeColours {
+  'default' =  'default',
+  'warning' = 'warning',
+  'teal' = 'teal',
+}
+
+type KoboRangeProps = {
+  max: number,
+  value: number,
+  /** uses time display for all required values */
+  isTime?: boolean,
+  /** assumes input is clickable and with move to where it is clicked */
+  onChange?: Function,
+  /** defaults to $kobo-blue */
+  color?: string,
+  /** optional string to append to max */
+  maxUnits?: string,
+  /** optional string to append to value */
+  currentUnits?: string,
```

Comment:
I think we should have just `units` for both places, as it's highly unlikely they would be different

Reply:
Also if I go with `isTime`, would that mean that the unit is known?